.classpath

```diff
@@ -1,41 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
To jest plik generowany przez eclipse. Bardzo dobrze, że go usunąłeś - nie potrzebujemy go w repozytorium.